Terminal.Gui/Views/ComboBox.cs

```diff
@@ -40,8 +40,8 @@ public class ComboBox : View {
 		/// </summary>
 		public ComboBox () : base()
 		{
-			search = new TextField ("") { LayoutStyle = LayoutStyle.Computed };
-			listview = new ListView () { LayoutStyle = LayoutStyle.Computed, CanFocus = true /* why? */ };
+			search = new TextField ("");
+			listview = new ListView () { LayoutStyle = LayoutStyle.Computed, CanFocus = true };
```

Comment:
I noticed you do not expose the `Source` Property of `ListView`.

Won't users expect a combobox to have the same API as `Listview` when dealing with the list source?